package-lock.json

```diff
@@ -709,7 +709,7 @@
       "dev": true
     },
     "design-system": {
-      "version": "git+https://github.com/Wikia/design-system.git#d95dc989abe023ff37c6e7db7084bb8aada52570",
+      "version": "github:Wikia/design-system#74af7bea40956ca45054bb0f4a7ab3017e255650",
```

Comment:
does it mean that on oasis and mobile-wiki design system is loaded twice? If yes, is there a possibility to check if design system is loaded already and not load it for the second time?